docker-compose.yml

```diff
@@ -80,3 +96,27 @@ volumes:
       o: bind
       type: none
       device: "${OTOPS_PATH_DEPLOYMENT_ELASTIC_SEARCH}/data"
+  chconf:
```

Comment:
There's an actual difference between mounting the host paths as that, paths, and doing it as docker volumes. This seems to be important for elastic search (it didn't work with the path mapping) but, apparently, Clickhouse doesn't mind.
Nice refactoring though!